README.md

```diff
@@ -23,6 +23,12 @@ Outside of the UCG repo, much of the team's discussion happens [in the `#t-opsem
 
 [zulip]: https://rust-lang.zulipchat.com/#narrow/stream/136281-t-opsem
 
+### Meetings
+
+The team holds weekly meetings on jitsi: <https://meet.jit.si/ucg-rust>. The meeting is currently
+scheduled for noon, Boston time. You can see what time that is in your timezone
```

Comment:
This mentions weekly at noon, but no day of the week?

Reply:
oops